pxtsim/simdriver.ts

```diff
@@ -367,6 +369,7 @@ namespace pxsim {
                                 this.startFrame(messageFrame);
                             }
                         } else {
+                            isDeferrableBroadcastMessage = true;
```

Comment:
How come this doesn't need to be in the start second simulator condition? Are two messages sent on the initial radio send?

Reply:
if you modify the program I wrote to send two radio messages it would do so -- it could take a moment for the frame to start up so passing it through the case where it'll check whether it's been loaded before sending feels right to me~